cmake/modules/hunter_pkgconfig_export_target.cmake

```diff
@@ -29,15 +29,25 @@ function(hunter_pkgconfig_export_target PKG_CONFIG_MODULE)
   endif()
   add_library("${target_name}" INTERFACE IMPORTED GLOBAL)
 
+  if (HUNTER_BUILD_SHARED_LIBS)
```

Comment:
I think using `if(BUILD_SHARED_LIBS)` should work fine here. Have to work for both `HUNTER_BUILD_SHARED_LIBS=ON` and `hunter_config(... CMAKE_ARGS BUILD_SHARED_LIBS=ON)`.

Reply:
Yeah I wasn't really sure what to do with this, thanks.